### ISSUE TITLE
refile does not work with rack (~> 2.0)

### ISSUE THREAD

**Issue by ck2000** [refile/refile#522]
I have added the folllowing to my GemFile:

gem "refile", "0.6.2", require: "refile/rails"

Bundler could not find compatible versions for gem "rack":
  In snapshot (Gemfile.lock):
    rack (= 2.0.1)

  In Gemfile:
    bootstrap-datepicker-rails was resolved to 1.6.4.1, which depends on
      railties (>= 3.0) was resolved to 5.0.1, which depends on
        actionpack (= 5.0.1) was resolved to 5.0.1, which depends on
          rack (~> 2.0)

    refile (= 0.6.2) was resolved to 0.6.2, which depends on
      sinatra (~> 1.4.5) was resolved to 1.4.6, which depends on
        rack-protection (~> 1.4) was resolved to 1.5.3, which depends on
          rack

    refile (= 0.6.2) was resolved to 0.6.2, which depends on
      sinatra (~> 1.4.5) was resolved to 1.4.6, which depends on
        rack (~> 1.4)

Could you please fix this dependency?

**Comment by netqyq** [comment]
when I add 
```
# Upload
gem "refile", require: "refile/rails", github: "refile/refile"
gem "refile-mini_magick"
```

```
Resolving dependencies...
Bundler could not find compatible versions for gem "rack":
  In Gemfile:
    rails (~> 5.0.1) was resolved to 5.0.1, which depends on
      actionpack (= 5.0.1) was resolved to 5.0.1, which depends on
        rack (~> 2.0)

    refile-mini_magick was resolved to 0.1.0, which depends on
      refile (~> 0.5) was resolved to 0.6.2, which depends on
        sinatra (>= 1.4.5) was resolved to 1.4.5, which depends on
          rack (~> 1.4)

    sass-rails (~> 5.0) was resolved to 5.0.6, which depends on
      sprockets (< 4.0, >= 2.8) was resolved to 3.7.1, which depends on
        rack (< 3, > 1)

    devise was resolved to 4.2.0, which depends on
      warden (~> 1.2.3) was resolved to 1.2.7, which depends on
        rack (>= 1.0)

    devise was resolved to 4.2.0, which depends on
      warden (~> 1.2.3) was resolved to 1.2.7, which depends on
        rack (>= 1.0)
```

**Comment by sobrinho** [comment]
See #529
